README.md

```diff
@@ -26,7 +27,26 @@ install](http://hoa-project.net/Source.html).
 
 ## Quick usage
 
-…
+Once installed, commands from libraries can be run with the following command
+line pattern:
+
+```sh
+$ hoa <library-name>:<command-name> <options> <inputs>
+```
+
+Running `hoa` with no argument will list all the available commands with a small
+description. Note: If the option `--no-verbose` is present, the list of commands
+will not be formatted. Thus, used in conjunction with Zsh resources present in
+[`Hoa\Devtools`](http://central.hoa-project.net/Resource/Library/Devtools/Resource/Zsh), you
```

Comment:
Why not moving these resources to the Cli library ? 

Requiring another lib just to get the completion for the current lib seems weird or is it just me ?